Jumpscale/clients/nbh_vertex/NBHClient.py

```diff
@@ -0,0 +1,424 @@
+import requests
```

Comment:
i think jumpscale wraps it using j.clients.http

Reply:
I don't see anything there regarding sessions